packages/grafana-data/src/types/logs.ts

```diff
@@ -31,6 +31,7 @@ export enum LogsMetaKind {
   Number,
   String,
   LabelsMap,
+  Error,
```

Comment:
Added so we can have special styling for errors - e.g. red color.

Reply:
Can we add some comment what does LogsMetaKind mean? I was a bit confused whether it is some log type or what's the difference from log level.